### PR TITLE
[risk=low][RW-7008] Include usernames in most emails sent to users

### PR DESCRIPTION
**Pull request by jmthibault79**
Add usernames to 4 emails sent by the RW: free credits warning and expiration and AAR warning and expiration.

Before and after examples can be seen here: https://docs.google.com/document/d/1ksfIb9fgJMETz4yVA25TGafzX7-ddg7ES-Fm0pdbjcA/edit# 

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [x] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [x] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Comment by jmthibault79** [comment]
Proposals and questions here: https://docs.google.com/document/d/1ksfIb9fgJMETz4yVA25TGafzX7-ddg7ES-Fm0pdbjcA/edit#

**Comment by jmthibault79** [comment]
Not clear whether we want to add the username to the instructions email.  We can do that here or in a different PR if so

File: api/src/main/resources/emails/dollarthresholdemail/content.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
 <body>
 <img src="${HEADER_IMG}">
 <div>Hello ${FIRST_NAME} ${LAST_NAME},</div>
-<div>This is a reminder that you have used ${USED_CREDITS} in free credits offered by the All of Us Program.
+<div>This is a reminder that your user ${USERNAME} has used ${USED_CREDITS} in free credits offered by the All of Us Program.
```
Comment (calbach) [inline api/src/main/resources/emails/dollarthresholdemail/content.html:32]: wording nit: would "your account" be more appropriate here? In our system "user" is really an abstract representation of the person receiving the email, so from their perspective they might consider this to be their "account".

**Review by calbach — APPROVED** [review]

File: api/src/main/resources/emails/dollarthresholdemail/content.html (hunk 1 above)
Comment (jmthibault79) [reply api/src/main/resources/emails/dollarthresholdemail/content.html:32]: makes sense - I'll check with the UX review crew

Comment (jmthibault79) [reply api/src/main/resources/emails/dollarthresholdemail/content.html]: done

**Review by calbach — APPROVED** [review]

**Pull request merged**